cypress/integration/orders/orders.PoLineFilters.spec.js

```diff
@@ -0,0 +1,105 @@
+import NewOrder from '../../support/fragments/orders/newOrder';
+import TestType from '../../support/dictionary/testTypes';
+import Orders from '../../support/fragments/orders/orders';
+import TopMenu from '../../support/fragments/topMenu';
+import OrdersHelper from '../../support/fragments/orders/ordersHelper';
+import basicOrderLine from '../../support/fragments/orders/basicOrderLine';
+import getRandomPostfix from '../../support/utils/stringTools';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import DateTools from '../../support/utils/dateTools';
+
+describe('orders: Test Po line filters', () => {
+  const today = new Date();
+  const subcriptionDate = DateTools.getFormattedDate({ date: today }, 'MM/DD/YYYY');
+  const order = { ...NewOrder.defaultOrder };
+  const orderLine = {
+    ...basicOrderLine.defaultOrderLine,
+    details: {
+      productIds: [{
+        productId: '9781868885015',
+        productIdType: '8261054f-be78-422d-bd51-4ed9f33c3422'
+      }],
+      subscriptionFrom: `${DateTools.getFormattedDate({ date: today }, 'YYYY-MM-DD')}T00:00:00.000+00:00`,
+      subscriptionInterval: 0
+    },
+    donor: `Autotest donor_${getRandomPostfix()}`,
+    publisher: `Autotest Publishing_${getRandomPostfix()}`,
+    requester: `Autotest requester_${getRandomPostfix()}`,
+    selector: `Autotest selector_${getRandomPostfix()}`,
+    fundDistribution: [{
+      code: 'USHIST',
+      fundId: '65032151-39a5-4cef-8810-5350eb316300',
+      distributionType: 'percentage',
+      value: 100
+    }],
+    physical: {
+      createInventory: 'Instance, Holding, Item',
+      materialType: '',
+      materialSupplier: '',
+      volumes: ['test vol. 1']
+    },
+    vendorDetail: {
+      instructions: `autotest instructions_${getRandomPostfix()}`,
+      noteFromVendor: `autotest note_${getRandomPostfix()}`,
+      referenceNumbers: [
+        {
+          refNumber: '123456-78',
+          refNumberType: 'Vendor title number',
+          vendorDetailsSource: 'OrderLine'
+        }
+      ],
+      vendorAccount: '8910-10'
+    },
+  };
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  let orderLineNumber;
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getOrganizationApi({ query: 'name="Amazon.com"' })
+      .then(organization => {
+        order.vendor = organization.id;
+        orderLine.physical.materialSupplier = organization.id;
+        orderLine.eresource.accessProvider = organization.id;
+      });
+    cy.getLocations({ query: `name="${OrdersHelper.mainLibraryLocation}"` })
```

Comment:
I remember that on bugfest environment we didn't have mainLibraryLocation
It seems like possible future problem. It is nesessary to use this location?